src/services/review/RepoCheckerService.ts

```diff
@@ -189,8 +243,12 @@ export default class RepoCheckerService {
       .mapErr((error) => new SiteCheckerError(`${error}`))
   }
 
-  remover(repo: string, repoExists: boolean) {
-    if (repoExists) {
+  // This function removes the recently cloned repo. This function should only be called in non-prod environments
+  remover(repo: string) {
```

Comment:
When's this function used?

Reply:
anywhere but prod. 
enables running of this in dev mode + testing by the team in staging.